### ISSUE TITLE
Add possibility to give permissions to share dashboards/cards to group

### ISSUE THREAD

**Issue by moumny** [metabase/metabase#4333]
Hi guys,

I played around with the sharable links feature and it looks great !

To make make more use of it, it would be best if we could give to a trusted user the ability to share links.
For example some users have only access to non sensitive data and such can freely share.

Thats always better that sharing through CSV/excel :)

**Comment by farzadshafiee** [comment]
Right now, the only way to give sharing permissions is to give make a user admin.
I believe adding a separate permission for question sharing would be safer. If users can create an artifact and are trusted to share questions, then they should be able to do so without having administrative privileges.
The back and forth for sharing between the IT and other departments got so annoying and counterproductive, that we ended up giving admin access to at least one person in each department.

**Comment by flamber** [comment]
Related to #6799

**Comment by briansboyd** [comment]
Without this permission, the admin has to come in and OK each and every question to be shared publicly.
As Metabase gets more popular within our organization, this will clearly become burdensome, and put the unhealthy pressure to make everyone an admin.
Is it really that difficult to simply create this extra permission that allows a user to share a public link to a question?

**Comment by jamiew** [comment]
Yes, I would very much like to grant our Power Users or even Default users the ability to share any given query or dashboard

**Comment by daveram** [comment]
Adding the use case that we have data owners within our org who decide where data gets shared, vs admins who run Metabase and the associated application database.  We've tried giving the data owners Settings access in application permissions, but that allows them to turn public sharing off/on for the entire Metabase instance, not turn public sharing on a dashboard off/on.  Admins don't generally know what data needs to be shared in larger organizations, so it harms larger adoption of Metbase to have them as the only group able to enable the public links

**Comment by albertoperdomo** [comment]
Hello, I am a product manager at Metabase and currently looking after sharing and embedding.

For those of you who have upvoted this issue or expressed the need:
I would love to learn more about your use case and gather a bit more of context around this feature request.

Feel free to drop a few lines on this GH issue outlining the context (maybe sharing in this context is sharing outside of the org in an email or Slack message, or perhaps it's embedding in a website, e.g. blog post). 

If you prefer, you can also schedule some time to chat with me over Zoom: https://calendly.com/alberto-metabase/30min

I am looking forward to hearing from you.

**Comment by TTiole** [comment]
@albertoperdomo 
In our case, we have a workflow on our application's backend which allows our account managers to override a specific dashboard for a customer (or group of customers) which is embedded in our application.
In other words:
We have a default metabase dashboard ID we use for all customers
We allow overriding this dashboard ID for customers or groups of customers
Our application dynamically fetches the appropriate dashboard and creates an embeddable link for our application

I'd like account managers to be able to self-serve, as they know what their customers need better than anyone, thus they most fit to build their custom dashboards. Waiting on an admin to sign off on this creates unnecessary friction, we're a fully remote team and there's not always going to be someone online to help. However, I'm not particularly fond of giving everyone administrator privileges either. Though that's what I'm ending up having to do.

**Comment by adeleh-t** [comment]
How would this be implemented (in case someone wants to help with a PR)?
I assume the permission needs to be added at group-level. But where does this setting can naturally reside?

![Screenshot 2024-06-30 at 13-26-02 Permissions · Admin · Metabase](https://github.com/metabase/metabase/assets/107904879/962be260-0678-43ab-a53c-3f529873d788)
